common-app-lib/src/cmr/common_app/api/request_context_user_augmenter.clj

```diff
@@ -85,7 +84,7 @@
   It expects the request context is already associated with the request."
   [handler]
   (fn [request]
-    (let [{:keys [request-context]} request]
+    (let [{:keys [_request-context]} request]
```

Comment:
You don't need the let statement.  Can you please remove it.

Reply:
Ok, I see I was too conservative, fixing all these now. Thank you